crates/oxc_ast/src/serialize.rs

```diff
@@ -634,6 +634,35 @@ impl ESTree for FormalParametersRest<'_, '_> {
     }
 }
 
+struct FormalParameterItem<'a, 'b>(&'b FormalParameter<'a>);
+
+impl ESTree for FormalParameterItem<'_, '_> {
+    fn serialize<S: Serializer>(&self, serializer: S) {
+        let param = self.0;
+
+        if S::INCLUDE_TS_FIELDS && param.has_modifier() {
+            let mut state = serializer.serialize_struct();
+            state.serialize_field("type", &JsonSafeString("TSParameterProperty"));
+            state.serialize_field("start", &param.span.start);
+            state.serialize_field("end", &param.span.end);
+            if let Some(accessibility) = &param.accessibility {
+                state.serialize_field("accessibility", accessibility);
+            } else {
+                state.serialize_field("accessibility", &Null(()));
+            }
```

Comment:
This can be simplified a little. The serializer for `Option<T>` already includes the logic to convert `None` to `null`.

```suggestion
            state.serialize_field("accessibility", &param.accessibility);
```